tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -597,3 +597,6 @@ def test_physical_couter_reset_aarch64(uvm_nano):
             reg_id, reg_value = parts
             if reg_id == cntpct_el0:
                 assert int(reg_value, 16) < max_value
+                break
+    else:
+        raise Exception("Did not find CNTPCT_EL0 register in snapshot")
```

Comment:
nit: I'm surprised pylint doesn't complain, but I think  `RuntimeError` is a better fit than bare `Exception`.

Reply:
changed